src/reports/reports.controller.ts

```diff
@@ -0,0 +1,46 @@
+import { Controller, Get, Param, Res } from '@nestjs/common';
+import { ReportsService } from './reports.service';
+import { Response } from 'express';
+
+@Controller('reports')
+export class ReportsController {
+  constructor(private readonly reportsService: ReportsService) {}
+  // Trae a los pacientes
+  @Get('prueba')
+  async obtenerDatos() {
+    return await this.reportsService.trayendoDatos();
+  }
+  // Trae el pdf con la hoja de registro del paciente
+  @Get('bill/:patientId')
```

Comment:
Endpoint should be taking the consumptionSheetId as identifier, not the patientId, please modify that and bring all info based on that, also please write all the code in english.